csrc/python_frontend/fusion_record.h

```diff
@@ -932,6 +933,87 @@ struct CastOpRecord : RecordFunctor {
   PrimDataType dtype_;
 };
 
+struct CatOpRecord : RecordFunctor {
+  CatOpRecord(
+      std::vector<State> _args,
+      std::vector<State> _outputs,
+      int64_t dim)
+      : RecordFunctor(
+            std::move(_args),
+            std::move(_outputs),
+            "ops.cat",
+            RecordType::CatOp),
+        dim_(dim) {}
+  virtual ~CatOpRecord() = default;
+  virtual RecordFunctor* clone() final {
+    return new CatOpRecord(*this);
+  }
+
+  virtual size_t hash() const final {
```

Comment:
nitpick: Please follow Kevin's example and put a quick note on the hash key for this, (even though it's only trivia here with a single dim :laughing: